hil/cli/misc.py

```diff
@@ -0,0 +1,109 @@
+"""Miscellaneous commands go here"""
+import click
+from hil.cli.client_setup import setup_http_client
+
+C = setup_http_client()
+
+
+@click.command(name='serve', help='Run a development HIL server')
+@click.argument('port', type=click.IntRange(0, 2**16-1),
```

Comment:
I think it would be better for this to be an option `--port`, rather than a positional argument.